client/src/components/HealthCard/HealthCard.jsx

```diff
@@ -0,0 +1,265 @@
+import React, { useState, useEffect } from "react";
+import useStyles from "./styles";
+import {
+  Card,
+  CardContent,
+  Typography,
+  Divider,
+  IconButton,
+  List,
+  ListItem,
+} from "@material-ui/core";
+import VisibilityIcon from "@material-ui/icons/Visibility";
+import logo from "../../../images/mitre-logo.svg";
+
+var QRCode = require('qrcode')
+
+function HealthCard(props) {
+  const styles = useStyles();
+  const { fhirBundle } = props;
+
+  const [healthCard, setHealthCard] = useState({
+    patient: {
+      full_name: "",
+      birth_date: "",
+    },
+    immunizations: [],
+    qrCodes: [],
+  });
+  const [showDateOfBirth, setShowDateOfBirth] = useState(false);
+  const [dateOfBirthDisplay, setDateOfBirthDisplay] = useState("**/**/****");
+  const [qrCodesRendered, setQrCodesRendered] = useState([]);
+
+  useEffect(() => {
+    const generateQR = async (text, index) => {
+      try {
+        return await QRCode.toDataURL(text, {type: 'svg', errorCorrectionLevel: 'low', version: 22 });
+      } catch (err) {
+        console.error(err)
+      }
+    }
+
+    fetch("health_card", {
+      method: "POST",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/fhir+json",
+      },
+      body: JSON.stringify(sampleData),
+    })
+      .then((response) => response.json())
+      .then((responseJson) => {
+        setHealthCard(responseJson);
+      
+        Promise.all(
+          responseJson.qrCodes.map(async (item, i) => {
+            return await generateQR(item, i);
+          })
+        ).then(qrs => {
+          setQrCodesRendered(qrs);
+        })        
+      })
+      .catch(console.log());
+  }, []);
+
+  const handleClickShowDateOfBirth = () => {
+    setShowDateOfBirth(!showDateOfBirth);
+    if (!showDateOfBirth) {
+      setDateOfBirthDisplay("**/**/****");
+    } else {
+      setDateOfBirthDisplay(healthCard.patient.birth_date);
+    }
+  };
+
+  return (
+    <div className={styles.healthCard}>
+      <Card className={styles.card}>
+        <CardContent className={styles.cardContent}>
+          <div className={styles.flexRow}>
```

Comment:
I can update to use top level Box instead of div, but the classes were all autogenerated from the sketch file. If we re-generate them from an updated design later, much easier to update them in a separate styles js than migrate one by one to element attributes.